lib/src/Base/Optim/openturns/MultiStart.hxx

```diff
@@ -46,7 +47,8 @@ public:
 
   /** Constructor with parameters */
   MultiStart(const OptimizationAlgorithm & solver,
-             const Sample & startingPoints);
+             const Sample & startingPoints,
+             const Bool keepResults = ResourceMap::GetAsBool("MultiStart-KeepResults"));
```

Comment:
I dont like such flags being added to ctors because it prevents extending the api.
I'd prefer if we could just provide an accessor for it.